src/renderers/dom/client/eventPlugins/SimpleEventPlugin.js

```diff
@@ -48,8 +57,8 @@ var invariant = require('invariant');
  *   'topAbort': { sameConfig }
  * };
  */
-var eventTypes = {};
-var topLevelEventsToDispatchConfig = {};
+var eventTypes: {[key: string]: DispatchConfig} = {};
+var topLevelEventsToDispatchConfig: {[key: TopLevelTypes]: DispatchConfig} = {};
```

Comment:
Nice! I realized that flow didn't automatically find out about this type and couldn't think of a way to to make it do so. I didn't think about adding the type in the declaration :)